.github/workflows/pr-request-review.yml

```diff
@@ -0,0 +1,29 @@
+name: Request Review
+
+on:
+  issue_comment:
+    types:
+      - created
+
+jobs:
+  request-review:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    if: github.repository_owner == 'cryostatio' && github.event.issue.pull_request && startsWith(github.event.comment.body, '/request_review')
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        
+      - name: Add label and request review
+        run: |
+          reviewers="cryostatio/reviewers"
```

Comment:
I think having this as a separate "constant" makes sense, in which case maybe it may as well be extracted to an `env:` section in the workflow?